src/routes/index.ts

```diff
@@ -234,6 +235,14 @@ router.post(config.OVERSEAS_ENTITY_QUERY_URL, authentication, ...validator.overs
 router.get(config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL, authentication, confirmOverseasEntityDetails.get);
 router.post(config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL, authentication, confirmOverseasEntityDetails.post);
 
+router.route(config.OVERSEAS_ENTITY_PRESENTER_URL)
+  .all(
+    authentication,
+    navigation.hasOverseasEntity,
```

Comment:
Strictly we should be checking that the details have been confirmed? That would require a new session key though. Probably ok to just check for entity number. Even that seems overkill when just asking for an email address.